clappr/clappr-ios
# Reload Container

# Goal

- Create reload flow on Container
- Clear objects when stop is call
- Expose function on Playbutton
- Fix spinner visibility when pause and stop events is triggered

# How to test

- Try to call reload function with other configurations, like: 

On `Player.swift#150`

```swift
open func reload(sourceUrl: String) {
    core?.activeContainer?.reload(source: sourceUrl)
}
```

On `ViewController.swift#101`

```swift
@IBAction func recreatePlayer(_ sender: Any) {
    player.destroy()
    player = nil
    createPlayer()
        
    let sourceUrl = "https://www.radiantmediaplayer.com/media/bbb-360p.mp4"
    player.reload(sourceUrl: sourceUrl)
    player.play()
}
```

- The other goals were covered by unit tests. You can check there
